### DISCUSSION TITLE
I am not able to find my build file when i build my flutter app through codemagic.yaml file in publishing section saying no artifacts found

### DISCUSSION THREAD

**Discussion by Sharafat-dev000** [codemagic-ci-cd/codemagic-docs#2622] [Q&A]
### If you are looking for help, please confirm the following...

- [x] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

 i build my flutter app through codemagic.yaml file

### Expected results

i am expected to get build file but i don't find any after successful built from codemagic.yaml file

### Actual results

after successful built i don't get any build file and in publishing section displaying logs that no artifacts found when i try to built from codemagic.yaml file

### Build id (optional)

6684492e44e02cd85608a6e9

**Comment by dtrdic** [comment]
Hello @Sharafat-dev000, there are two different ways to build your app: using the Workflow editor and codemagic.yaml. However, you can't combine configurations from both methods. It looks like you initially added your app intending to use the Workflow editor, so let's stick to that and comment out the codemagic.yaml file for now. You can switch to using codemagic.yaml later if you prefer.

Also let me share with you [this step-by-step](https://labs.codemagic.io/your-first-flutter-app-to-appstore/#3) guide which will help you set up and codesign your app corectly

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
